docs/interpreter/pig.md

```diff
@@ -51,6 +53,20 @@ group: manual
 
     HADOOP\_CONF\_DIR and TEZ\_CONF\_DIR needs to be specified in `ZEPPELIN_HOME/conf/zeppelin-env.sh`.
 
+- Spark Local Mode
+    
+    Nothing needs to be done for spark local mode
+    
+- Spark Mode
+    
+    For now, only yarn-client mode is supported.  To enable it, you need to set property SPARK_MASTER to yarn-client
+    and set SPARK_JAR to the spark assembly file uploaded to hdfs.
```

Comment:
so a higher level question - why 2 mode: Spark Local and Spark? why not have `zeppelin.pig.execType` as `spark` and then look for `SPARK_MASTER` or `zeppelin.pig.spark.master` or `spark.master`? That seems like more like how people have understand how to use Spark.